src/main/webapp/components/workspace/workspace.js

```diff
@@ -87,6 +91,7 @@ const queryToSearch = query => {
         sortOrder: sort.substring(splitIndex + 1, sort.length),
       }
     }),
+    ...pagingFields,
```

Comment:
It appears that a search that sets pageSize to zero runs faster than one that only returns the status, not sure why